examples/change_notifier_custom_state/lib/main.dart

```diff
@@ -145,21 +139,19 @@ class BooksLocation extends BeamLocation<BooksState> {
   BooksLocation(RouteInformation routeInformation) : super(routeInformation);
 
   @override
-  BooksState createState(RouteInformation routeInformation) =>
-      BooksState().fromRouteInformation(routeInformation);
+  BooksState createState(RouteInformation routeInformation) {
+    final state = BooksState().fromRouteInformation(routeInformation)
+                  ..addListener(notifyListeners);
+    return state;
+  }
+     
```

Comment:
In the case of beaming back, the `initState` is called before `createState` so the newly created state has no listeners. I was not sure how it can be solved better, so I just moved adding a listener to `createState` method.

Reply:
@jmysliv Ah, I see... great catch.
This is a bug in `beamBack` that I  think I fixed right now in https://github.com/slovnicki/beamer/commit/d3981bc8b086fed3b20173f990de5289c3ee31d3

So we can go back to adding listeners just in `initState` :slightly_smiling_face: